### PR TITLE
Change Gold Block TPs

### PR DESCRIPTION
**Pull request by Diet-Cola**
Jump = Go up
Sneak = Go down

Thanks to Papers PlayerJumpEvent we can make this a lot more like FTB mods and also more intuitive imo

Making this PR prematurely so still need to test, will ping when tested and yup this will conflict with my other PR #38

### CODE REVIEW

File: src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameFeatures.java [hunk 1]
```diff
@@ -542,33 +544,29 @@ public void onPlayerDeath(PlayerDeathEvent event) {
...
 				return;
 			}
 		}
-		for(int y = 0; y < block.getY(); y++) {
-			if (doTeleport(block, event.getPlayer(), y)) {
+		event.getPlayer().sendMessage(ChatColor.RED + "No suitable destination was found");
+	}
+
+	@EventHandler(priority = EventPriority.HIGH, ignoreCancelled = true)
+	public void goldBlockSneak(PlayerToggleSneakEvent event) {
```
Comment (Ameliorate) [inline src/main/java/com/programmerdan/minecraft/simpleadminhacks/hacks/GameFeatures.java:563]: You should check if the player is enabling or disabling their sneak. If they're disabling it, you should return. If you don't, one press of shift will teleport the player twice.

**Comment by wingzero54** [comment]
This is current and works, right Diet? I saw Devoted implemented this a couple days ago

**Comment by wingzero54** [comment]
@Maxopoly  really looking forward to this one, makes elevators drastically more intuitive

**Comment by TheOrangeWizard** [comment]
what if we had this but with respawn anchors instead

just kidding

unless...

**Comment by wingzero54** [comment]
There's a PR floating around somewhere to make them respawn anchors, but at this point it kinda seems like it would just mess up everybody's existing elevators for little reason

**Comment by TheOrangeWizard** [comment]
They look way cooler and don't overlap with acid blocks which i think is more reason than we normally have for messing up existing stuff

**Pull request merged**